configs/sklearn/linear_model.json

```diff
@@ -0,0 +1,126 @@
+{
+    "PARAMETERS_SETS": {
+        "implementations": {
+            "algorithm": [
+                { "library": "sklearn", "device": "cpu" },
+                { "library": "sklearnex", "device": "cpu" }
+            ]
+        },
+        "linear implementations": {
+            "algorithm": [
+                { "library": "sklearn", "device": "cpu" },
+                { "library": "sklearnex", "device": "cpu" },
+                { "library": "sklearnex.preview", "device": ["cpu", "gpu"] }
```

Comment:
status of preview has changed, preview for LinearRegression no longer exists (same for ensemble)

Reply:
Current preview status was applied for configs.